.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: 🐛 Bug
+
+description: Report an issue to help improve the project.
+
+labels: ["🛠 goal: fix"]
+
+body:
```

Comment:
the body should contain
- description (✅ you have included)
- steps to reproduce (🔍 missing item)
- expected behavior (🔍 missing item)
- screenshots  (✅ you have included)
- user device (only applicable if bug if specific to certain devices only) (🔍 missing item)
- additional info (optional)  (✅ you have included)